tests/e2e/test_rosacli_cluster.go

```diff
@@ -3067,3 +3063,105 @@ var _ = Describe("Sts cluster creation supplemental testing",
 				Expect(clusterID).ToNot(BeNil())
 			})
 	})
+
+var _ = Describe("User can set availability zones to create rosa multi-az STS cluster supplemental testing",
+	labels.Feature.Cluster,
+	func() {
+		defer GinkgoRecover()
+
+		var (
+			rosaClient     *rosacli.Client
+			clusterService rosacli.ClusterService
+
+			customProfile      *profilehandler.Profile
+			clusterID          string
+			ocmResourceService rosacli.OCMResourceService
+			testingClusterName string
+		)
+		BeforeEach(func() {
+
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+			clusterService = rosaClient.Cluster
+			ocmResourceService = rosaClient.OCMResource
+
+			By("Prepare custom profile")
+			customProfile = &profilehandler.Profile{
+				ClusterConfig: &profilehandler.ClusterConfig{
+					HCP:           false,
+					MultiAZ:       true,
+					STS:           true,
+					OIDCConfig:    "",
+					NetworkingSet: false,
+					BYOVPC:        false,
+					Zones:         "us-west-2a,us-west-2b,us-west-2c",
+				},
+				AccountRoleConfig: &profilehandler.AccountRoleConfig{
+					Path:               "/aa/bb/",
+					PermissionBoundary: "",
+				},
+				Version:      "latest",
+				ChannelGroup: "candidate",
+				Region:       "us-west-2",
+			}
+			customProfile.NamePrefix = constants.DefaultNamePrefix
+
```

Comment:
Set Name prefix in case level to make it related to case id. Otherwise leak resource cannot be traced back.

Reply:
Updated using case id as name prefix.